pom.xml

```diff
@@ -103,7 +103,7 @@
     <timestamp>${maven.build.timestamp}</timestamp>
     <maven.build.timestamp.format>yyyy-MM-dd HH:mm</maven.build.timestamp.format>
 
-    <beam.version>0.6.0</beam.version>
+    <beam.version>2.0.0</beam.version>
 
     <dataflow.release_name>Google Cloud Dataflow SDK for Java</dataflow.release_name>
     <dataflow.container_version>${project.version}-20170317</dataflow.container_version>
```

Comment:
For the code at HEAD with this PR to work, we would need a new (dated) container.

Options:
* build a new 2.0.0-beta4-SNAPSHOT-20170522 container for this PR only.
* deal with breakage on the assumption that maven release:prepare is going to immediately change the version to (2.0.0-SNAPSHOT) or (2.1.0-SNAPSHOT) and `HEAD` will be broken for a short time.